python/cudf/cudf/__init__.py

```diff
@@ -18,7 +18,6 @@
     register_series_accessor,
 )
 from cudf.core.scalar import (
```

Comment:
This can be a single line now, but the trailing comma prevents isort from making this refactor. I would do it by hand.
```suggestion
from cudf.core.scalar import Scalar
```